edc-extensions/bpn-validation/bpn-validation-core/src/test/java/org/eclipse/tractusx/edc/validation/businesspartner/functions/BusinessPartnerGroupFunctionTest.java

```diff
@@ -82,14 +85,14 @@ void setUp() {
     void evaluate_noParticipantAgentOnContext() {
         reset(context);
         assertThat(function.evaluate(EQ, "test-group", createPermission(EQ, List.of()), context)).isFalse();
-        verify(context).reportProblem(eq("ParticipantAgent not found on PolicyContext"));
+        verify(context).reportProblem("ParticipantAgent not found on PolicyContext");
```

Comment:
please don't just change unrelated code. Supplying a matcher is good practice IMO, because it increases expressivity.